web/packages/extension/assets/popup.html

```diff
@@ -34,6 +34,7 @@
             </div>
         </div>
         <div class="buttons-container">
+            <button class="hidden" id="permissions-button">Permanently enable for this site</button>
```

Comment:
I wonder if we should highlight this button a bit more, so it's at least more prominent than the "open SWF player" button?

Reply:
This can be a follow-up. I'm not good with design. I did move this button to a separate `buttons-container` div, so that it ends up on a separate line explicitly rather than just by a quirk of the CSS.